### ISSUE TITLE
'Invite by link' includes captcha when disabled in user signup settings

### ISSUE THREAD

**Issue by dantefromhell** [humhub/humhub#6332]
During the `Invite by link` signup flow the user has to solve a captcha, even if the
`Include captcha in registration form` option in `User administration` -> `Settings` is disabled.

### What steps will reproduce the problem?
- Allow Invite by link
- Disable `Include captcha in registration form` option
- Copy & Paste the invite link for a space into a new browser
- Captcha field shows after entering an email address

### What is the expected result?
Captcha field is not shown.

### What do you get instead?
Captcha field is shown.

### Additional info

| Q                | A
| ---------------- | ---
| HumHub version   | 1.14.2
| PHP version      | 8.1
| Operating system |

**Comment by gevorgmansuryan** [comment]
I not able to reproduce

![Image](https://github.com/humhub/humhub/assets/11891855/09373dd8-2e3d-42bd-87c7-0db14fc0fb61)

**Comment by gevorgmansuryan** [comment]
@dantefromhell can You please provide some screenshoot/screencast ?

**Comment by dantefromhell** [comment]
Thanks @gevorgmansuryan 

When trying to create the screenshot for you I managed to find what looks like the underlying issue.

Although I have disabled the captcha in my settings I realized that the setting is still shows as `enabled` when I reload the setting page.

Looking at the 'Developer Tools' in my Browser I can see it sends `AuthenticationSettingsForm[showCaptureInRegisterForm]: 0` to the `https://portal.radiantiris.org/admin/authentication` endpoint, but the configuration change does not get applied/ stored.
Using `yii settings/list` I can confirm that `auth.showCaptureInRegisterForm` remains at value `1`.

It feels logical to me to investigate this issue first, as it might be the source of my original problem.

Any hints how I can debug this?

**Comment by dantefromhell** [comment]
Disabling the capture via the `yii` command
```
yii settings/set user auth.showCaptureInRegisterForm 0
```
resulting in the signup page working as expected.

After disabling the captcha setting via `yii` I tried to re-*enable* via the Administration menu and this fails too.

The correct title for this issue might be 'captcha setting not updated from web UI'.

This behaviour is also reproducible in HumHub 1.14.2 (I updated my initial post)
